package.json

```diff
@@ -54,7 +54,7 @@
     "trailingComma": "none"
   },
   "resolutions": {
-    "h3": "^1.8.0-rc.2",
+    "h3": "npm:h3-nightly@latest",
```

Comment:
Please use normal releases as RC. nightly is not stable nor has anything new

Reply:
Will do as soon as I get to the bottom of this. I've been able to deploy this succesfully on cloudflare pages, so now I'm looking for possible regression on the `vercel-edge` preset.

Reply:
Should this not be merged @pi0?

@Hebilicious if you open an issue with some details I can send it around internally

Reply:
It is in, this is an outdated comment 👍🏽

Reply:
We just switched to nightly in last nitro edge (yesterday). LGTM just please make sure latest lockfile is in place.